admin_guide/cluster_capacity.adoc

```diff
@@ -143,6 +143,12 @@ $ oc adm policy add-cluster-role-to-user cluster-capacity-role \
     system:serviceaccount:default:cluster-capacity-sa
 ----
 
+[NOTE]
+====
+The `default` specification in the above command refers to the `default`
+project. If using an alternative project, amend the command accordingly.
+====
+
```

Comment:
Actually, I'd change this note into an annotation on the command.

```
----
$ oc adm policy add-cluster-role-to-user cluster-capacity-role \
    system:serviceaccount:default:cluster-capacity-sa <1>
----
<1> If the service account is not in the `default` project, replace `default` with the project name.
```

Reply:
Thanks @kalexand-rh. I have pushed those changes and updated the preview also.